db/seeds.rb

```diff
@@ -24,10 +24,10 @@
 Doctor.create(doctors)
 
 patients = [
-	{ name: 'Tony Hawk', description: 'Fracture Skull', dob: Date.parse('12/05/1951'), gender: 'Male', phone: 9841123456,address: 'Nepal'  },
-	{ name: 'Brett Lee', description: 'Stomach Problem', dob: Date.parse('1/12/1950'), gender: 'Male', phone: 9841098456, address: 'India'},
-	{ name: 'John', description: 'Headache', dob: Date.parse('1/2/1960'), gender: 'Male', phone: 9841126812, address: 'China' },
-	{ name: 'Ravindra', description: 'Mental Patient', dob: Date.parse('10/2/1956'), gender: 'Male', phone: 9841790456, address: 'Japan' }
+	{ name: 'Tony Hawk', description: 'Fracture Skull', dob: Date.parse('12/05/1951'), gender: 'Male', phone: 9841123456,address: 'Nepal',email_id: 'tony@test.com'},
```

Comment:
space after comma. Always use the same convention/style. Even if you decide to use only yours style, do not mix it. This applies for spaces, braces, quotes. You can use rubocop for this - https://github.com/bbatsov/rubocop